### ISSUE TITLE
Background image and image issues

### ISSUE THREAD

**Issue by JordanTYC** [jonbhanson/flutter_native_splash#329]
**Describe the bug**
The background image displays an uni-color background instead (see screenshot) and my center image does not appear.
- I've checked the android files, images are generated inside.
- I use flutter_native_splash.yaml file.
- I have an initialization, so I put dependency in the dependencies.

**Screenshots**
https://imgur.com/a/wL044cr   ->  first image is what I have, and second is the image that I call.
On emulator, I have a white screen instead (before I had a green one also, but I cannot reproduce it)

**Device (please complete the following information):**
 - Device: Samsung Galaxy note10 Lite, and also emulator
 - OS: Android 12

**Flutter doctor**
[√] Flutter (Channel stable, 2.10.3, on Microsoft Windows [version 10.0.19041.264], locale fr-FR)
    • Flutter version 2.10.3 at C:\Users\AY025039\Documents\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 7e9793dee1 (5 weeks ago), 2022-03-02 11:23:12 -0600
    • Engine revision bd539267b4
    • Dart version 2.16.1
    • DevTools version 2.9.2

[√] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
    • Android SDK at C:\Users\AY025039\AppData\Local\Android\sdk
    • Platform android-32, build-tools 32.1.0-rc1
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.1.1)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.1.32228.430
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2021.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)

[√] Connected device (5 available)
    • SM N770F (mobile)            • RF8N30WW4NR   • android-arm64  • Android 12 (API 31)
    • sdk gphone64 x86 64 (mobile) • emulator-5554 • android-x64    • Android 12 (API 32) (emulator)
    • Windows (desktop)            • windows       • windows-x64    • Microsoft Windows [version 10.0.19041.264]
    • Chrome (web)                 • chrome        • web-javascript • Google Chrome 100.0.4896.75
    • Edge (web)                   • edge          • web-javascript • Microsoft Edge 99.0.1150.30

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

**Comment by jonbhanson** [comment]
I see you are using Android 12.  Did you configure the `android_12:` section of the config?

Please post the `flutter_native_splash` portion of your config.

**Comment by JordanTYC** [comment]
`flutter_native_splash:

  background_image: "assets/login.png"

  android_12:

  image: "assets/log.png"`

**Comment by JordanTYC** [comment]
I've tried image in and out of 'android_12'

**Comment by vikt0r40** [comment]
I have absolutely the same problem. For iOS is working perfectly for Android 12 I see just a solid color. Firstly I was seeing just white screen on my device but then I modify Values-v31/styles.xml and I started to see just the color of the background image. I tried everything in flutter_native_splash with no luck

**Comment by jonbhanson** [comment]
Are you launching from Android Studio?  As noted in the [FAQ](The splash screen may not appear when you launch the app from Android Studio. However, it should appear when you launch by clicking on the launch icon in Android.), the splash screen may not appear when you launch the app from Android Studio. However, it should appear when you launch by clicking on the launch icon in Android.

**Comment by JordanTYC** [comment]
Yes, I use Android Studio and the splash screen appears by clicking on the launch icon in Android. But sadly after reading your FAQ, background image is not supported by Android 12.

Btw, do you have an example (code) of an initialization (ie [https://pub.dev/packages/flutter_native_splash#3-set-up-app-initialization-optional](url)) + a FutureBuilder in the build method ? Because in my current code, my device displays the splash screen + an another screen which appears when snapshot has no data yet. And I would like to delete this second screen because my data is supposed to be loaded after initialization.

**Comment by jonbhanson** [comment]
You may find this [example](https://pub.dev/packages/flutter_native_splash/example) helpful.

**Comment by JordanTYC** [comment]
In my initialization, I instantiate a Future, and when it's done, I build this FutureBuilder (instead of the Scaffold in your example) :


FutureBuilder<MyType>(
future: futureMetadata,
            builder: (BuildContext context, AsyncSnapshot<MyType> snapshot) {
              if (snapshot.hasData) {
                return MyWidget(data: snapshot.data);
              } else {
                return const ErrorScreen();
              }

The problem is that the splash screen is displayed, then the ErrorScreen (few milliseconds) before building MyWidget. I just want the splash screen, no more screen.

**Comment by jonbhanson** [comment]
This is more of a general Flutter question than a problem specific to this package.  I would suggest that you post it to [https://stackoverflow.com/](https://stackoverflow.com/).
